vllm/engine/llm_engine.py

```diff
@@ -1224,7 +1224,7 @@ def update_prefill_num_computed_tokens(
                               skip)
 
             # Tracing
-            self.do_tracing(scheduler_outputs)
+            self.do_tracing(scheduler_outputs, finished_before)
```

Comment:
Should I also pass the `skip` list in addition to `finished_before`, and exclude the sequence groups marked in this list during tracing?